app/assets/javascript/application.js

```diff
@@ -25,3 +25,34 @@ Rails.start()
 // const imagePath = (name) => images(name, true)
 
 initAll()
+
+document.addEventListener('DOMContentLoaded', function () {
+  setUpEventListeners()
+})
+
+document.addEventListener('turbo:render', function () {
+  setUpEventListeners()
+})
+
+const setUpEventListeners = () => {
+  document.querySelectorAll('.search-form-toggle').forEach((toggle) => {
+    toggle.addEventListener('click', () => {
+      document.querySelectorAll('.search-form').forEach((element) => {
```

Comment:
How many form `.search-form` are there?
if it's just one we could use `querySelector` instead of `querySelectorAll`